assets/js/spa-app.js

```diff
@@ -8,6 +8,7 @@ var _typeof12 = typeof Symbol === "function" && _typeof13(Symbol.iterator) === "
     return obj && typeof Symbol === "function" && obj.constructor === Symbol && obj !== Symbol.prototype ? "symbol" : typeof obj === "undefined" ? "undefined" : _typeof13(obj);
 };
 
+
```

Comment:
Why the addition of the new line?

Reply:
Removed the space not sure why it got added. It grunt concat product so it could have been added in when the beta was created. That space should be removed when we build the plugin for release.